components/switch/switch-mixin.js

```diff
@@ -50,7 +50,7 @@ export const SwitchMixin = superclass => class extends RtlMixin(FocusVisiblePoly
 			}
 
 			.d2l-switch-container {
-				background-color: #ffffff;
+				background-color: var(--d2l-switch-container-background-color, '#fff');
```

Comment:
Could you add a basic visual diff test to exercise this variable? Just so it can fail if we ever accidentally break this.

Reply:
I don't think we need to document it though since we don't really want folks to know about it. 🤫